### PR TITLE
Ensure that RSSI is not casted to an int

### PR DESCRIPTION
**Pull request by bastbu**
# PR for issue #1129 

## What is being addressed

Ensures that RSSI is not casted to an int.

### CODE REVIEW

**Comment by danigian** [comment]
What about LoraKeysManagerFacade\FunctionsBundler\FunctionBundlerRequest.cs and related logic for ComputePreferredGateway?

**Review by danigian — CHANGES_REQUESTED** [review]
What about LoraKeysManagerFacade\FunctionsBundler\FunctionBundlerRequest.cs and related logic for ComputePreferredGateway?

File: LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/LoRaPhysical/Rxpk.cs [hunk 1]
```diff
@@ -47,7 +47,7 @@ public class Rxpk
         public string Codr { get; set; }
 
         [JsonProperty("rssi")]
-        public int Rssi { get; set; }
+        public double Rssi { get; set; }
```
Comment (danigian) [inline LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/LoRaPhysical/Rxpk.cs:50]: Please revert this change as it's not needed and will be removed anyways with #1153

**Review by danigian — APPROVED** [review]
Looks good to me now. Please run a full ci before merging

File: LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/LoRaPhysical/Rxpk.cs (hunk 1 above)
Comment (bastbu) [reply LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/LoRaPhysical/Rxpk.cs]: Done in aab1bb53029445d5dfe49891416cf0b458224419
